### PR TITLE
docs: cheatsheet section in readme

### PR DESCRIPTION
**Pull request by dreamfliper**
I hope this makes the newcomers feel more friendly. I was struggling to find these short cuts.

### CODE REVIEW

**Comment by dreamfliper** [comment]
Sorry I didn't see any comment from you on files, but I guess it's about the caps? And sorry again about deleting the tail spaces, it's done automatically by VS Code.

File: README.md [hunk 1]
```diff
@@ -101,12 +103,42 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
...
 
+### Cheatsheet
+Draft Note:
+- `oo`: Append to scratchpad
+- `oo`: Open to scratchpad
+
+Create Note:
+- `on`: New note
+
+Wroksapce:
```
Comment (chrisgrieser) [inline README.md:114]: typo ("Workspace")

File: README.md [hunk 2]
```diff
@@ -101,12 +103,42 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
...
+- `ow`: Worksapce switcher
```
Comment (chrisgrieser) [inline README.md:115]: typo

File: README.md [hunk 3]
```diff
@@ -101,12 +103,42 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
...
+- `ov`: Vault switcher
+
+Search:
+- `oe`: Search all external link via vault
+- `ot`: Search tags
+- `ob`: Search bookmarked file/searches
+- `or`: Search recent files
+- `oa`: Open file (like images)
+- `op`: Search plugins/themes
+- o`,:` Search settings and installed plugins
```
Comment (chrisgrieser) [inline README.md:125]: typo: o`,:`

File: README.md [hunk 4]
```diff
@@ -117,7 +149,7 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
 ## About the Developer
 In my day job, I am a sociologist studying the social mechanisms underlying the digital economy. For my PhD project, I investigate the governance of the app economy and how software ecosystems manage the tension between innovation and compatibility. If you are interested in this subject, feel free to get in touch.
 
-__Profiles__
```
Comment (chrisgrieser) [inline README.md:120]: These are double-trailing spaces, which are meaningful in markdown, please restore them ("two-space-rule")

File: README.md [hunk 5]
```diff
@@ -128,7 +160,7 @@ __Profiles__
 
 *Note that for questions, bug reports, or feature requests for this workflow, please open a [GitHub issue](https://github.com/chrisgrieser/shimmering-obsidian/issues), since it is better suited for technical support.*
 
-__Donate__
```
Comment (chrisgrieser) [inline README.md:131]: here as well

File: README.md [hunk 6]
```diff
@@ -101,12 +103,42 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
...
+- `oo`: Open to scratchpad
+
+Create Note:
+- `on`: New note
+
+Wroksapce:
+- `ow`: Worksapce switcher
+- `ov`: Vault switcher
+
+Search:
```
Comment (chrisgrieser) [inline README.md:118]: The main search `o` should also be listed here I guess ;)

File: README.md [hunk 7]
```diff
@@ -101,12 +103,42 @@ Alfred Workflow with dozens of features for controlling your [Obsidian vault](ht
...
+- `oe`: Search all external link via vault
+- `ot`: Search tags
+- `ob`: Search bookmarked file/searches
+- `or`: Search recent files
+- `oa`: Open file (like images)
```
Comment (chrisgrieser) [inline README.md:123]: Better phrase it "Open attachments", since then `oa` as mnemonic becomes clear

**Review by chrisgrieser — CHANGES_REQUESTED** [review]

**Comment by chrisgrieser** [comment]
Oh, sorry, I forget to confirm the review 🙈

Have sent it now

**Comment by dreamfliper** [comment]
It should all fixed

**Comment by chrisgrieser** [comment]
LGTM! Thanks for the PR

**Pull request merged**